src/System.Text.Json/src/System/Text/Json/Serialization/ReadStackFrame.cs

```diff
@@ -14,6 +14,11 @@ internal struct ReadStackFrame
         internal object ReturnValue;
         internal JsonClassInfo JsonClassInfo;
 
+        // Support Dictionary
+        internal List<string> TempDictKeys;
```

Comment:
As mentioned earlier I think we can remove the TempDictKeys and the other two fields below.